### ISSUE TITLE
Table Validator radio button group missing label

### ISSUE THREAD

**Issue by nschonni** [wet-boew/wet-boew#8329]
Running aXe on http://wet-boew.github.io/wet-boew/demos/tablevalidator/tablevalidator-en.html
It is flagging that the radio button group doesn't have the same aria-labledby
```
To solve this violation, you need to:
Fix at least one (1) of these issues:

    All elements with the name "access" do not reference the same element with aria-labelledby

    Fieldset contains unrelated inputs
    Related nodes:

#idprefix
#uniqueprefix
```

Not sure about this one, but just flagging it for review
